test/integration/azureService.js

```diff
@@ -0,0 +1,27 @@
+const chai = require('chai');
+const azureService = require('../../app/lib/azureService');
+
+const expect = chai.expect;
+
+describe('Azure Service', () => {
+  it('should upload file to azure', function test(done) {
+    this.timeout(5000);
+    const name = 'test-file.json';
+    azureService.uploadToAzure('test/resources/test-file.json', name)
```

Comment:
I would be good to delete the file as part of the test teardown. The subsequent test for checking if there are more than 0 files is most likely going to pass always. I'm not sure what the point of it is.